ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.sass

```diff
@@ -0,0 +1,61 @@
+@import '../../../../styles/fonts-constants'
+@import '../../../../styles/colors'
+
+.form
+  height: 100%
+  display: flex
+.center
+  margin: auto
+  text-align: center
+.trips-modal-container
+  position: relative
+.trips-title
+  text-align: center
+  margin-bottom: 0
+.form-row
+  display: flex
+  justify-content: center
+  align-items: center
+.trips
+  height: 23rem
+  overflow-x: hidden
+  margin: 0.5rem
+  display: flex
+  flex-direction: column
+  position: relative
+  .trips-user
+    border: 1px solid mat-color($primary, A100)
+    display: flex
+    justify-content: space-around
+    align-items: center
+    width: 100%
+    .profile-photo
+      font-size: $display-2
+    .remove-icon
+      font-size: $title
+      display: table
+      vertical-align: middle
+      color: $dark-red
+    .trips-list
+      font-size: $button
+      margin-top: 1rem
+      width: 12rem
+.trips-btn
+  display: flex
+  justify-content: center
+.trip-btn
```

Comment:
There is no this style in trips-modal.component.html

Reply:
fixed